infra/cron/src/scheduler.js

```diff
@@ -100,3 +101,13 @@ growthUpdateCampaignsQueue.add(
   { repeat: { cron: '30 20 * * *' } }
 )
 logger.info('Scheduled growthUpdateCampaigns job.')
+
+// Growth campaign update job. Runs daily at 20:30UTC (~12:30 PST).
```

Comment:
Typo: it's the consistency checker, not campaign update job. Also time is not correct. should be 23:30UTC